cmd/generator/main.go

```diff
@@ -154,6 +156,7 @@ var generatorCmd = &cobra.Command{
 	},
 }
 
+// getReleasePlugins returns the slice of plugins sorted by plugin version, descending.
```

Comment:
nit: this reads more as a description of the input/output parameters, and doesn't tell me what it actually /does/.

```suggestion
// getReleasePlugins queries GitHub for all releases of the given plugin, sorting by plugin versioning descending.
```